spec/controllers/api/encryptables_controller_spec.rb

```diff
@@ -776,10 +775,38 @@
       get :show, params: { id: encryptable_file.id }, xhr: true
     end
 
-    it 'displays transferred encryptable and dont download it' do
+    it 'download transferred encryptable new encryption algorithm' do
+      login_as(:bob)
+
+      encryptable_file = prepare_transferred_encryptable(Crypto::Symmetric::Aes256iv)
+
+      login_as(:alice)
+
+      expect(controller).to receive(:send_file).exactly(:once)
+
+      get :show, params: { id: encryptable_file.id }, xhr: true
+    end
+
+    it 'displays transferred encryptable and dont download it old encryption algorithm' do
+      login_as(:bob)
+
+      encryptable_file = prepare_transferred_encryptable(Crypto::Symmetric::Aes256)
+
+      login_as(:alice)
+
+      expect(controller).not_to receive(:send_file)
+      expect_any_instance_of(CrudController).to receive(:render_entry).exactly(:once)
+
+      get :show, params: { id: encryptable_file.id,
+                           encryptable: ActionController::Parameters.new({
+                                                                           test: 1
+                                                                         }).permit! }, xhr: true
```

Comment:
permit? why is this used here? can't see any reason why we should permit attributes in a controller spec

Reply:
I changed it, but it was already there from the old tests